### ISSUE TITLE
Put greek letters as tick labels

### ISSUE THREAD

**Issue by Koushikphy** [veusz/veusz#329]
I want to put tick labes as greek letters like &pi;, 2&pi; etc. But with the current version I can't do put tick labes like `\pi', it throws error
```
Error in expression '\pi': Unable to parse file: unexpected character after line continuation character (<string>, line 1)
```
How do I do this?

**Comment by ikost** [comment]
Hi, I am using v 3.0.1.1 and can not reproduce the problem. To add tick labels you have to create a dataset for labels (\pi without Apostrophs). Please find example attached.
[greek_ticks.zip](https://github.com/veusz/veusz/files/3656735/greek_ticks.zip)

**Comment by jeremysanders** [comment]
Is this in a script you wrote? Remember in Python strings, \ needs to be written as \\, so you might need '\\pi'

**Comment by Koushikphy** [comment]
I've just started with veusz and don't know know much about it, can you guide through how do I create the tick labels from the start?

**Comment by ikost** [comment]
Have you checked my example? Is it something you are looking for?

**Comment by Koushikphy** [comment]
I have hundreds of data and I can't figure out how to put labes in just 3-4 places on x axis

**Comment by ikost** [comment]
As soon as you are not replying my question, let me guess what you mean. It is not vary straightforward to add labeled axis in Veusz but it is generally allowed. Let me imagine few options:
1. You have a numerical axis and you want to label a few ticks. The easiest way is to add manually a label nere those ticks. You can bound the label to axis value if you plan to update the axis boundaries.
2. You want labelled axis. First you create X and Y axis widgets. Set the axis setting (where you need labels) to "label" type. Plot data with XY widget (you need some dummy numerical dataset "behind" labels anyway) and specify a dataset for the labels on the needed axis. The dummy dataset (like X=1,2,3,4 in my example ) is needed for the Veusz to know the positions of the points but is then overwritten by the dataset with labels. Hope this helps.

**Comment by Koushikphy** [comment]
Your method also put labels over the data points or am I getting it wrong? Here's a sample data https://file.io/igSDH9 can you plot this with x axis labels at 0,pi/2, pi, 3pi/2 and 2pi ?

**Comment by jeremysanders** [comment]
Without using labels, an alternative is to change on the axis the tick label format setting to something like `%Vg \pi` (`%Vg` means the Veusz general format). This adds a pi after each normal numerical label. You can then multiply your data by 1/pi so that pi is plotted as pi, by entering `1/pi` in the scale setting on that axis.

**Comment by Koushikphy** [comment]
yeah but then it plots like '0 pi', '1 pi' etc.. which looks ugly

**Comment by Koushikphy** [comment]
Can any of you just plot the above data (https://a.uguu.se/d8xmDRZUbzsN.dat) with x axis labels at 0,&pi;/2, &pi;, &pi;/2 and 2&pi; and upload the `.vsz` file? that would be a great help.

**Comment by ikost** [comment]
> Your method also put labels over the data points or am I getting it wrong? Here's a sample data https://a.uguu.se/d8xmDRZUbzsN.dat can you plot this with x axis labels at 0,pi/2, pi, 3pi/2 and 2pi ?

Yes, it does. You can hide those labels in the setting of the XY widget ("hide" checkbox  in the label tab "xy").

**Comment by ikost** [comment]
> Can any of you just plot the above data (https://a.uguu.se/d8xmDRZUbzsN.dat) with x axis labels at 0,π/2, π, π/2 and 2π and upload the `.vsz` file? that would be a great help.

Here we go. Please find the file attached. To do the plot, I have done the following: 
1. imported your dataset
2. created the new dataset with the expression: "rad"="degrees_value_from_imported_dataset"/180
3. plotted your Y values vs new rad X values. It gives you the 0, 0.5, 1, 1.5, 2 on X axis. This can be tuned by the settings of the ticks.
4. Changed in  X axis setting -->"1.2" tab for Tick labels--> Format = %Vg \pi (as Jeremy suggested). This has added \pi after each tick.
[ticks_v2.zip](https://github.com/veusz/veusz/files/3658335/ticks_v2.zip)
